lib/src/next/v1/nu_router.dart

```diff
@@ -12,6 +12,8 @@ import '../../typings.dart';
 /// Extend to create your NuRoute. Contains the configuration of a Route that is
 /// going to be presented in a [Nuvigator] by the [NuRouter]
 abstract class NuRoute<T extends NuRouter, A extends Object, R extends Object> {
+  NuRoute({Map<String, dynamic> metaData}) : metaData = metaData ?? {};
```

Comment:
it's not possible to use like that:
```suggestion
  NuRoute({this.metaData = const {}});
```
because of the `const` it would not be possible to add data later, and Dart does not allow to set a map default value that is not a const:
`The default value of an optional parameter must be constant.`